google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/test_helpers/env/CloudEnv.java

```diff
@@ -239,6 +241,7 @@ private void configureUserAgent(EnhancedBigtableStubSettings.Builder stubSetting
         // nothing special
         break;
       case REQUIRE_CFE:
```

Comment:
to confirm, is this expected to change the settings for the `REQUIRE_CFE` case?

Reply:
yep